src/engine/enginefilteriir.h

```diff
@@ -102,7 +102,7 @@ class EngineFilterIIR : public EngineFilterIIRBase {
             char* desc;
             FidFilter* filt = fid_design(spec_d, sampleRate, freq0, freq1, adj, &desc);
             int delay = fid_calc_delay(filt);
-            qDebug() << QString().fromAscii(desc) << "delay:" << delay;
+            qDebug() << QString().fromLatin1(desc) << "delay:" << delay;
```

Comment:
I have just noticed that we need to free "desc" after every fid_design call 
https://uazu.net/fidlib/fidlib.txt 
but this can be fixed in a separate PR. I will file a bug https://bugs.launchpad.net/mixxx/+bug/1797978